extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/ClientExceptionMapper.java

```diff
@@ -0,0 +1,19 @@
+package io.quarkus.rest.client.reactive;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import javax.ws.rs.Priorities;
+
+//TODO: add
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.METHOD)
+public @interface ClientExceptionMapper {
```

Comment:
Given one of the main annotations is called `RegisterRestClient`, I wonder if it should be `RestClientExceptionMapper`? I won't fight for it and maybe we have other annotations with just `Client` but if not, maybe it's good to be consistent and explicit?

Reply:
I'll leave it up to @michalszynkiewicz and @radcortez to decide this one

Reply:
@michalszynkiewicz and @radcortez quick feedback welcome as it would be a nice addition for CR1. Thanks!

Reply:
So, on the REST Client API we have:

- `@RestClient`
- `@RegisterRestClient`
- `@RegisterClientHeaders`
- `@RegisterProvider`
- `@RegisterProviders`
- `@ClientHeaderParam`
- `@ClientHeaderParams`

No real consistency :)

I think we should include `Register`, since it is used more often. Maybe `@RegisterClientExceptionMapper`? or just `@RegisterExceptionMapper`?

Reply:
@radcortez @geoand  @michalszynkiewicz  This is a new feature so either you folks can agree on a name or it probably won't be in 2.7.

Reply:
I vote to leave it as is

Reply:
I'll vote for `@RegisterExceptionMapper` for the sole reason that it will popup on the autocomplete when you are adding any of the other annotations, so better chance for users to discover it (and it is shorter than `@RegisterClientExceptionMapper`).

Reply:
> I vote to leave it as is

Sounds fair :)

Reply:
Register doesn't make much sense to me here. `@RestClientExceptionMapper` would be more consistent with what we have but without Rest it's shorter. I'm okay with it leaving as it is